include/env.h

```diff
@@ -17,6 +17,8 @@ extern int env_list_setenv(struct ul_env_list *ls, int overwrite);
 extern void env_list_free(struct ul_env_list *ls);
 extern struct ul_env_list *env_list_from_fd(int pid);
 
+extern struct ul_env_list *env_list_add(struct ul_env_list *ls, const char *str);
```

Comment:
This naming is very muddy. `env_list_import[_from_current]()`?

Reply:
OK